source/deploy/consider-mobile-vpn-options.rst

```diff
@@ -20,12 +20,12 @@ A Virtual Private Network (VPN) allows a device outside a firewall to access con
 
 We recommend one of two options: `per-app VPN <#id3>`_ or a `device VPN <#id4>`_ to secure your deployment. Both options are compatible with most EMM providers. 
 
-We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/mobile/mobile-appstore-install.html>`__ and to review the following commonly-asked questions about data security on mobile devices:
+We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/deploy/consider-mobile-vpn-options.html?highlight=mobile%20vpn#consider-mobile-vpn-options>`__ and to review the following commonly-asked questions about data security on mobile devices:
```

Comment:
```suggestion
We also recommend following our `recommended steps to secure your deployment <https://docs.mattermost.com/deploy/consider-mobile-vpn-options.html>`__ and to review the following commonly-asked questions about data security on mobile devices:
```

Reply:
It's always best to have the URL without the appended search terms. :)

Reply:
Okay, This is a valuable addition. Thanks a lot.